src/argon2/_password_hasher.py

```diff
@@ -26,10 +27,11 @@ class PasswordHasher(object):
     r"""
     High level class to hash passwords with sensible defaults.
 
-    Uses *always* Argon2\ **i** and a random salt_.
+    Uses *always* Argon2\ **id** and a random salt_ for hashing, but it can
```

Comment:
Suggestion: always uses, not uses always

Reply:
Gah I *always* get this one wrong.